packages/components/Molecules/Search/SearchResultsV2.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div class="rpl-search-results">
+    <div v-if="error" class="rpl-search-results__error-msg">
+      <slot name="error">{{ error.message || errorMsg }}</slot>
+    </div>
+    <template v-else>
+      <div class="rpl-search-results__header">
+        <div class="rpl-search-results__info" v-if="range && count">
+          <slot name="count">
+            Displaying {{ range }} of {{ count }} results
+          </slot>
+        </div>
+        <div v-if="!!$slots.sort" class="rpl-search-results__sort">
+          <slot name="sort"></slot>
+        </div>
+      </div>
+      <template v-if="loading">
+        <slot name="loading">
+          ...loading
```

Comment:
TODO: Add a better default loading experience. Would like design input though.

Reply:
May need some UX research on this